packages/flutter_tools/lib/src/commands/assemble.dart

```diff
@@ -217,10 +217,10 @@ class AssembleCommand extends FlutterCommand {
     return false;
   }
 
-  late final Environment _environment = _createEnvironment();
+  late final Future<Environment> _environment = _createEnvironment();
 
   /// The environmental configuration for a build invocation.
-  Environment _createEnvironment() {
+  Future<Environment> _createEnvironment() async {
```

Comment:
Why did this become async?

Reply:
I think at some point while I developed this it needed something async - seems that is no longer the case.

Made sync again.